Source/AwsCommonRuntimeKit/mqtt/Mqtt5Client.swift

```diff
@@ -125,8 +125,8 @@ public typealias OnWebSocketHandshakeInterceptComplete = (HTTPRequestBase, Int32
 public typealias OnWebSocketHandshakeIntercept = (HTTPRequest, @escaping OnWebSocketHandshakeInterceptComplete) async -> Void
 
 // MARK: - Mqtt5 Client
-public class Mqtt5Client {
-    internal var clientCore: Mqtt5ClientCore
+public final class Mqtt5Client: Sendable {
+    internal let clientCore: Mqtt5ClientCore
```

Comment:
default visibility is internal https://docs.swift.org/swift-book/documentation/the-swift-programming-language/accesscontrol/#Default-Access-Levels. We should remove internal everywhere.